src/js/player.js

```diff
@@ -2672,6 +2673,62 @@ class Player extends Component {
     this.trigger('exitFullWindow');
   }
 
+  /**
+   * Handle a keydown event from any Component of this player which is not
+   *  handled by that Component to support player-wide hotkeys.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that was passed from a Component.
+   */
+  handleKeyPress(event) {
+
+    if (this.options_.hotkeys !== false) {
```

Comment:
We should do this check at the start of the function and return, that way we can eliminate this scope for the whole function.

Reply:
It would also be good to change this to the option named in the `dblclick` pull request: https://github.com/videojs/video.js/pull/5611#issuecomment-443266964 Although that could be done in a separate pull request if that works out better.

Reply:
@gkatsev I'm going to leave it as it is for now; having a method per key can be a separate PR.

Reply:
Sounds good, though, we should make sure on the direction we want to go with it so we don't ship with it as is if we wanted to have it per key.